ui/shared/DataListDisplay.tsx

```diff
@@ -13,8 +13,9 @@ type FilterProps = {
 type Props = {
   isError: boolean;
   items?: Array<unknown>;
-  emptyText: string;
+  emptyText: string | React.ReactNode;
```

Comment:
```suggestion
  emptyText: React.ReactNode;
```

`string` is also a `ReactNode`